src/components/places/places.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import styled from 'styled-components'
+import AddPlaceForm from './add-place-form/add-place-form'
+import MapWithAllPlaces from './map-with-all-places/map-with-all-places'
+import PlacesList from './places-list/places-list'
+import StyledInsideMapDivBig from '../common/inside-map-div-big'
+import { DEFAULT_MAP_PROPS } from '../../constants/constants'
+
+const StyledPlaces = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+
+const Places = ({ places, addPlace }) => {
+  return (
+
+    <StyledPlaces>
+      <AddPlaceForm onAddPlace={addPlace} places={places}/>
+      <MapWithAllPlaces 
+        places={places}
+        containerElement={<StyledInsideMapDivBig />}
+        {...DEFAULT_MAP_PROPS}
```

Comment:
Hmm trochę mało czytelne to rozwiązanie... Nie wiem czy nie lepiej zrobić swojego komponentu `Map`, który definiuje domyślne propsy.

Reply:
Nie do konca wiem jakby to mialo wygladac. Skoro mam rozne mapy, bo jedna jest z search boxem, inna jest z markerami. Inna bedzie jescze z directions, itd

Reply:
No wiem, ale możesz mieć komponent powiedzmy `RawMap`, który definiuje te domyślne propsy i wspólne zachowanie a potem komponent `MapWithPlaces`, będzie miał w sobie `RawMap` i nadpisywał to co ma nadpisywać

Reply:
hmm, bede myslec o tym bardziej jak sie na nowo wdroze w to co tu sie dzieje XD